jobs/bodhi-trigger.Jenkinsfile

```diff
@@ -0,0 +1,218 @@
+// Useful references:
+// - https://pagure.io/fedora-qa/fedora_openqa/blob/main/f/src/fedora_openqa/consumer.py
+// - https://pagure.io/fedora-ci/general/issue/436
+// - https://github.com/json-path/JsonPath
+// - https://sumiya.page/jpath.html
+
+// XXX: we should move this list to a separate YAML file for knob extensibility
+// in the future (for e.g. which tests to run for each)
+def srpms = [
+    'container-selinux',
+    'glibc',
+    'grub2',
+    'ignition',
+    'kernel',
+    'kexec-tools',
+    'NetworkManager',
+    'ostree',
+    'podman',
+    'rpm-ostree',
+    'rust-afterburn',
+    'rust-coreos-installer',
+    'rust-zincati',
+    'selinux-policy',
+    'systemd'
+]
```

Comment:
idea for future: we run a compose and `basic` tests for all packages we include in FCOS and for the list above we run the extended version (like we are doing here in this PR).

Reply:
Added as an XXX!